### PR TITLE
Sign in 

### PR DESCRIPTION
**Pull request by rodrieiz**
https://trello.com/c/NIB1cfPC/14-3-as-a-user-i-should-be-able-to-login-so-i-can-use-my-target-mvd-account

### CODE REVIEW

File: spec/requests/login_spec.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        {
+          email: user.email,
+          password: ''
+        }
+      end
+
+      before { post new_user_session_path, params: params, as: :json }
+
+      it 'error status unathorized' do
+        expect(response).to have_http_status(401)
```
Comment (enzofab91) [inline spec/requests/login_spec.rb]:
```suggestion
        expect(response).to be_bad_request
```

File: spec/requests/login_spec.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      {
+        email: user.email,
+        password: user.password
+      }
+    end
+
+    before { post new_user_session_path, params: params, as: :json }
+
+    it 'ok status' do
+      expect(response).to have_http_status(200)
```
Comment (enzofab91) [inline spec/requests/login_spec.rb]:
```suggestion
      expect(response).to be_successful
```

**Review by enzofab91 — APPROVED** [review]
Nice!

Left some minor comments.

File: spec/requests/login_spec.rb (hunk 1 above)
Comment (rodrieiz) [reply spec/requests/login_spec.rb]: Ok! Thanks!

Comment (rodrieiz) [reply spec/requests/login_spec.rb]: Do you mean "be_unauthorized"? It's works for me

**Review by DJA89 — APPROVED** [review]

File: spec/requests/login_spec.rb (hunk 1 above)
Comment (enzofab91) [reply spec/requests/login_spec.rb]: Yes!

**Pull request merged**